modules/openapi-generator/src/main/resources/typescript-fetch/package.mustache

```diff
@@ -6,7 +6,7 @@
   "main": "./dist/index.js",
```

Comment:
is this path still valid?

Reply:
Yes it is.

Reply:
but how can that be? i would expect all compiled files to be in `dist/src/`

Reply:
Compiled files are in `dist` folder.
Source files are in `src` folder.

Reply:
but the `tsconfig.json` does not explicitly configure `src` as the root directory, so I would expect something like `/src/index.ts` to be compiled into `/dist/src/index.js`

Reply:
I followed a real life example here: https://github.com/ccnokes/dom-event-utils
I guess the `src` for the root directory is "default" value so it doesn't have to be specified.

I've commited files that should fix the circleCI problem. Let me know if there is other problems! Thanks

Reply:
ok i tested it, I get `dist/index.js`